pkg/helm/io_test.go

```diff
@@ -0,0 +1,215 @@
+package helm
+
+import (
+	"os"
+	"reflect"
+	"testing"
+)
+
+func Test_Load(t *testing.T) {
+	tests := []struct {
+		name    string
+		content string
+		want    ValuesMap
+		err     bool
+	}{
+		{
+			name: "valid yaml file",
+			content: `
+replicaCount: 1
+nginx:
+  image: nginx
+  pullPolicy: IfNotPresent
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""`,
+			want: ValuesMap{
+				"replicaCount": 1,
+				"nginx": ValuesMap{
+					"image":      "nginx",
+					"pullPolicy": "IfNotPresent",
+				},
+				"imagePullSecrets": []interface{}{},
+				"nameOverride":     "",
+				"fullnameOverride": "",
+			},
+		},
+		{
+			name:    "empty yaml file",
+			content: ``,
+			want:    ValuesMap{},
+		},
+		{
+			name: "invalid yaml file",
```

Comment:
What's invalid about this YAML file, perhaps the test case name could say?

Reply:
```yaml
replicaCount: 1
image:
repository: nginx
```

not properly indented.

Reply:
That looks like properly indented YAML to me?

But it sounds like you've come up with a great name for the test case.

Reply:
shouldn't valid yaml will look like below? There is no proper indentation. 
```
replicaCount: 1
  image:
repository: nginx
```